packages/twenty-front/src/modules/object-metadata/hooks/useCreateOneObjectMetadataItem.ts

```diff
@@ -34,8 +33,20 @@ export const useCreateOneObjectMetadataItem = () => {
       variables: {
```

Comment:
the createOneObjectMetadataItem is awaited when called, we could try not await it before redirecting but this can be a problem if the component is unmounted.